components/feature/pwa/src/main/java/mozilla/components/feature/pwa/WebAppLauncherActivity.kt

```diff
@@ -44,11 +48,30 @@ class WebAppLauncherActivity : AppCompatActivity() {
         }
     }
 
+    override fun onResume() {
+        super.onResume()
+        backgroundEndTime = SystemClock.elapsedRealtimeNanos()
```

Comment:
Why not `foregroundStartTime/EndTime`?

Reply:
I thought it might be confusing if I was counting it as something like `timeSinceLastInForeground`. It seemed easier to count how long it's been in the background instead

Reply:
Ah, I see it now. I think the function should be renamed to `emitBackgroundTimingFact` to better align with the purpose and the "BACKGROUND_DURATION" enum.

Reply:
Good call! I changed it in the other places but I missed that one 😅